content/trailmap/prologue.md

```diff
@@ -0,0 +1,159 @@
++++
+title = "Dogecoin Trailmap: Prologue"
+date = "2021-12-10"
+type = "about"
+[ author ]
+name = "Dogecoin Foundation"
++++
+
+<section class="presentation">
+<div class="left">
+
+<div class="title">
+
+
+ ## <img title='You are here!' width="60px" style='display: inline;' src="/marker.png"/>Dogecoin Trailmap: Prologue
+
+<div class="underline"></div>
+</div>
+
+<div class="description">
+ 
+* [**Dogecoin Trailmap: Prologue**](/trailmap/prologue/) 
+* [Project: Libdogecoin](/trailmap/libdogecoin/)
+* [Project: Dogecoin Standard](/trailmap/standard/)
+* [Project: GigaWallet](/trailmap/gigawallet/)
+* Project: Dogecoin Keyring (App / SDK)
+* Task: Doge-In-A-Box (Core Containerisation)
```

Comment:
:heart:
```suggestion
* [Task: Doge-In-A-Box](https://github.com/dogecoin/docker/) (Core Containerisation)
```

Reply:
Oh sweet, I didn't know that was a thing (looks new!) I'll take it off the list, thanks!

Reply:
Looks awesome btw, can't wait to try it out :) hopefully an option to run without wallet and to point at the block store as params would be really handy

Reply:
Yeah, it's new and still in progress :) You may keep it to help/promote it :heart:  By the block store, you mean the  `-datadir` ? If yes, it's already possible. Take a look at the image documentation, only available as PR for now : https://github.com/dogecoin/docker/pull/13

I guess running without a wallet shouldn't be possible. Disabling is done through the `./configure` step, and we are using [release binaries](https://github.com/dogecoin/dogecoin/releases) which enable it by default. Need confirmation, but I think removing wallets may not be possible.

Reply:
Awesome, yeh I have my dev copy of core compiled without wallet support as I'm primarily interested in understanding how integrators (exchanges / platforms etc) would be using it, might not be a common use-case for others :sweat_smile:

Re: promoting it, there's a current effort by a bunch of people to make a fresh new dogecoin.com site and I believe that's supposed to have a whole section of projects relating to dogecoin so I think it would make sense to go there rather than in the Foundation roadmap?   If you're interested I can send you a discord invite to where folk are working on it :)

Reply:
Instead of doing the containerization, your trailmap bullet point could have been to help to finish containerization :) But however you feel, it was also to show that one of your goal is already in progress somewhere else if you weren't aware.

In any case, the support from the foundation could make sense for some situation, for example we would need someone/something trustable to manage Docker Hub credentials. Do you think it's the kind of thing the foundation could take care of ?

Reply:
> Hopefully an option to run without wallet

`dogecoind` has the option `-disablewallet` ;)